act-rules-format.bs

```diff
@@ -195,15 +216,16 @@ Note that atomic rules in a composite rule MAY have different applicability. Bec
 
 Expectations {#expectations}
 ==============================
-The expectations describe what the requirements are for the [test targets](#output-test-target) defined in the applicability section.
+
+The expectations describe what the requirements are for the [=test targets=] derived from the [applicability](#applicability).
 
 
 Expectations for Atomic Rules {#expectations-atomic}
 ----------------------------------------------------
 
-An [=atomic rule=] MUST contain one or more expectations. An expectation is an assertion that must be true about each [test target](#output-test-target) (see [Applicability](#applicability)). Each expectation MUST be distinct, unambiguous, and be written in plain language. Unlike in applicability, a certain level of subjectivity is allowed in expectations. Meaning that the expectation has only one possible meaning, but that meaning isn't fully quantifiable.
+An [=atomic rule=] MUST contain one or more expectations. An expectation is an assertion that must be true about each [=test target=] (see [Applicability](#applicability)). Each expectation MUST be distinct, unambiguous, and be written in plain language. Unlike in applicability, a certain level of subjectivity is allowed in expectations. Meaning that the expectation has only one possible meaning, but that meaning MAY not fully quantifiable.
 
-When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations are false, the test target `failed` the rule. If the atomic rule is used in a [=composed rule=], the composed rule may be `passed` when the atomic rule `failed`, depending on the [applicability](#applicability-composed) and [exxpectations-composed] of the composed rule.
+When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations are false, the test target `failed` the rule. If the atomic rule is used in a composed rule, the outcome of the composed rule may be `passed` when the atomic rule `failed`, depending on the [expectations](#expectations-composed) of the composed rule.
```

Comment:
"composed" -> "composite" (twice)
"may be" -> "can be" (drop the "may)

Suggestion: "If the atomic rule is used in a composed rule, the outcome of the composed rule may be `passed` when the atomic rule `failed`" -> "When atomic rules are used in composite rules, the outcome of the composite rules can be `passed` when the outcome of individual atomic rule is `failed`"